test/unit/math/fwd/prob/std_normal_test.cpp

```diff
@@ -0,0 +1,12 @@
+#include <stan/math/fwd/scal.hpp>
+#include <gtest/gtest.h>
+#include <vector>
+
+TEST(ProbAgradDistributionsStdNormal, fwd) {
+  using stan::math::fvar;
```

Comment:
These tests can be deleted if the expect_ad tests are added.

Reply:
Done.